addOns/ascanrules/src/main/resources/org/zaproxy/zap/extension/ascanrules/resources/Messages.properties

```diff
@@ -129,25 +127,13 @@ ascanrules.sqlinjection.alert.timebased.extrainfo=The query time is controllable
 ascanrules.sqlinjection.authbypass.name=SQL Injection - Authentication Bypass
 ascanrules.sqlinjection.authbypass.desc=SQL injection may be possible on a login page, potentially allowing the application's authentication mechanism to be bypassed 
 
-# TODO some duplicates
-
 ascanrules.sqlinjection.mysql.name=SQL Injection - MySQL
 ascanrules.sqlinjection.hypersonic.name=SQL Injection - Hypersonic SQL
 ascanrules.sqlinjection.oracle.name=SQL Injection - Oracle
 ascanrules.sqlinjection.postgres.name=SQL Injection - PostgreSQL
-ascanrules.sqlinjection.desc=SQL injection may be possible
-ascanrules.sqlinjection.soln=Do not trust client side input, even if there is client side validation in place. \nIn general, type check all data on the server side.\nIf the application uses JDBC, use PreparedStatement or CallableStatement, with parameters passed by '?'\nIf the application uses ASP, use ADO Command Objects with strong type checking and parameterized queries.\nIf database Stored Procedures can be used, use them.\nDo *not* concatenate strings into queries in the stored procedure, or use 'exec', 'exec immediate', or equivalent functionality!\nDo not create dynamic SQL queries using simple string concatenation.\nEscape all data received from the client.\nApply an 'allow list' of allowed characters, or a 'deny list' of disallowed characters in user input.\nApply the privilege of least privilege by using the least privileged database user possible.\nIn particular, avoid using the 'sa' or 'db-owner' database users. This does not eliminate SQL injection, but minimizes its impact.\nGrant the minimum database access that is necessary for the application.
```

Comment:
The spaces before the first newline should be removed (at least one to match the messages removed).